Error "A user authenticated as anonymous has attempted to access a session owned by user:customRealm/resAdmin" with standard and cluster

You get the following error when using Retrieve HTDS Description File and then Show HTDS Options and when you get this error, you have to close the browser if you want to continue to use the RES console:

Exception thrown by application class 'javax.faces.webapp.FacesServlet.service:154'
javax.servlet.ServletException: SESN0008E: A user authenticated as anonymous has attempted to access a session owned by user:customRealm/resAdmin.
at javax.faces.webapp.FacesServlet.service(FacesServlet.java:154)
at com.ibm.ws.webcontainer.servlet.ServletWrapper.service(ServletWrapper.java:1290)
at [internal classes]
at ilog.rules.res.console.ResponseHeaderFilter.doFilter(ResponseHeaderFilter.java:76)
at com.ibm.ws.webcontainer.filter.FilterInstanceWrapper.doFilter(FilterInstanceWrapper.java:207)
at [internal classes]
Caused by: com.ibm.websphere.servlet.session.UnauthorizedSessionRequestException: SESN0008E: A user authenticated as anonymous has attempted to access a session owned by user:customRealm/resAdmin.
at com.ibm.ws.webcontainer.session.impl.HttpSessionContextImpl.checkSecurity(HttpSessionContextImpl.java:751)
at [internal classes]
at org.apache.myfaces.context.servlet.SessionMap.getSession(SessionMap.java:75)
at org.apache.myfaces.context.servlet.SessionMap.getAttribute(SessionMap.java:46)
at org.apache.myfaces.context.servlet.AbstractAttributeMap.get(AbstractAttributeMap.java:91)
at org.apache.myfaces.portlet.PortletUtil.isPortletRequest(PortletUtil.java:64)
at org.apache.myfaces.lifecycle.RestoreViewExecutor.deriveViewId(RestoreViewExecutor.java:107)
at org.apache.myfaces.lifecycle.RestoreViewExecutor.execute(RestoreViewExecutor.java:57)
at org.apache.myfaces.lifecycle.LifecycleImpl.executePhase(LifecycleImpl.java:95)
at org.apache.myfaces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:70)
at javax.faces.webapp.FacesServlet.service(FacesServlet.java:139)
... 5 more

**How to reproduce:**
NB: The test was performed using Chrome. The error can be reproduced in the standard and cluster topologies.
- In the RES console, click Explorer.
- In the tree, select the miniloan ruleset.
- On the ruleset view, click Retrieve HTDS Description File.
- Select REST then click View. It opens a new tab next to the RES console tab in the browser.
- Click on the RES console tab in the browser, then click Show HTDS Options. You get an unauthorized exception "A user authenticated as anonymous has attempted to access a session owned by user:customRealm/resAdmin" and when you get this exception, you cannot use the RES console anymore. You have to close the browser and load the RES console again.

--- a/decisionserverruntime/config/server.xml
+++ b/decisionserverruntime/config/server.xml
@@ -22,4 +22,7 @@
 	<managedExecutorService jndiName="concurrent/drExecutorService"/>
 
 	<webContainer deferServletLoad="false" />
+	
+	<webAppSecurity useAuthenticationDataForUnprotectedResource="false" />
+	
 </server>

